ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/EngineConnectorConfigJvm.kt

```diff
@@ -23,22 +23,132 @@ public inline fun ApplicationEngineEnvironmentBuilder.sslConnector(
 /**
  * Mutable implementation of EngineSSLConnectorConfig for building connectors programmatically
  */
-public class EngineSSLConnectorBuilder(
-    override var keyStore: KeyStore,
-    override var keyAlias: String,
-    override var keyStorePassword: () -> CharArray,
+public actual class EngineSSLConnectorBuilder actual constructor() :
+    EngineConnectorBuilder(ConnectorType.HTTPS), EngineSSLConnectorConfig {
+    public constructor(
+        keyStore: KeyStore,
+        keyAlias: String,
+        keyStorePassword: () -> CharArray,
+        privateKeyPassword: () -> CharArray
+    ) : this() {
```

Comment:
we can make this constructor primary(with all `val` parameters) and make actual constructor secondary